mayastor/src/lvs/lvs_pool.rs

```diff
@@ -299,27 +300,21 @@ impl Lvs {
             });
         }
 
-        // this is a legacy argument, should not be used typically
-        if args.block_size != 512
-            && args.block_size != 4096
-            && args.block_size != 0
-        {
-            return Err(Error::Invalid {
-                source: Errno::EINVAL,
-                msg: format!(
-                    "invalid block size specified {}",
-                    args.block_size
-                ),
-            });
-        }
-
-        // fixup the device uri's to URL
+        // default to uring if kernel supports it
         let disks = args
             .disks
             .iter()
             .map(|d| {
-                if d.starts_with("/dev") {
-                    format!("aio://{}", d)
+                if Url::parse(d).is_err() {
```

Comment:
do we still need to support this "legacy"?

Reply:
This maintains the existing "auto" setting of io_if and allows existing yaml config files to work.